net-dns/libidn2/libidn2-2.0.4.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="Internationalized Domain Names (IDN) implementation"
+DESCRIPTION="Libidn2 is a free software implementation of IDNA2008, Punycode and TR46. Its \
+purpose is to encode and decode internationalized domain names."
+HOMEPAGE="https://gitlab.com/libidn/libidn2"
+COPYRIGHT="2011-2016 Simon Josefsson"
+LICENSE="GNU LGPL v3"
+REVISION="1"
+SOURCE_URI="http://ftp.gnu.org/gnu/libidn/libidn2-$portVersion.tar.gz"
+CHECKSUM_SHA256="644b6b03b285fb0ace02d241d59483d98bc462729d8bb3608d5cad5532f3d2f0"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+libVersion="0.3.3"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	libidn2$secondaryArchSuffix = $portVersion
+	lib:libidn2$secondaryArchSuffix = $libVersionCompat
+	lib:libgettextlib_$portVersion$secondaryArchSuffix
+	texinfo = $portVersion compat >= 4
```

Comment:
```diff
-	texinfo = $portVersion compat >= 4
+	cmd:idn2$secondaryArchSuffix = $portVersion
```
(I guess you wanted to put this in `BUILD_PREREQUIRES`, not in `PROVIDES`.)

BTW, it is recommended to sort the `lib:lib*` lines alphabetically.

Reply:
Do you mean `texinfo...` goes in `BUILD_PREREQUIRES`? I get a build error when I try to do that.

Reply:
No, I just dropped the whole `cmd:textinfo` line.

Reply:
BTW, I think I forgot to tell that the recipe should go in `haikuports/net-dns/libidn/`, not in `haikuports/net-dns/libidn2/`.

Reply:
I'm still getting a build error: POLICY ERROR: "/packaging/libidn2/bin/idn2" needs library "libintl.so.8", but the package doesn't seem to declare that as a requirement. I also get 2 other similar errors below that.

Reply:
That's strange. I'm almost sure I did not have this policy error when I first built libidn last wednesday. But now I have exactly the same error you get.
To fix this, you need to add a `lib:libintl$secondaryArchSuffix` to `REQUIRES` and a `devel:libintl$secondaryArchSuffix` to `BUILD_REQUIRES`.
Confirmed. It works as expected. (My build just finished.)

BTW, you also need to move the recipe to the right directory:
```
cd /boot/home/haikuports/net-dns
git mv libidn2/libidn2-2.0.4.recipe libidn
git commit libidn2/libidn2-2.0.4.recipe libidn/libidn2-2.0.4.recipe
```